test/fixtures/without-telemetry/input/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for app goes here",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "repository": "",
+  "scripts": {
+    "build": "ember build",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "ember test"
+  },
+  "devDependencies": {
+    "@ember/jquery": "^0.6.0",
+    "@ember/optional-features": "^0.7.0",
+    "bootstrap": "^4.3.1",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-ajax": "^5.0.0",
+    "ember-bootstrap": "^2.8.1",
+    "ember-cli": "~3.10.1",
```

Comment:
Lets use a more modern ember-cli and app blueprint (either 3.16 or 3.18)

Reply:
would those have any culry invocations to convert?

Reply:
Yes, ideally the modern version shouldn't have any curly invocations, @rwjblue do you still want to change that.

Reply:
Ya, I'd still prefer to use a new ember-cli. The templates that you check in aren't part of the app blueprint anyways so they would still have curlies or whatever you put in them.

Reply:
I think it would be a good idea if the checked in app used a ["mixed" layout](https://cli.emberjs.com/release/advanced-use/project-layouts/#classicorpods), i.e. a classic app w/ at least a couple of pods artifacts to test. Like maybe a pods route that invokes a pods component?

That way the tests will verify that #217 is resolved.

I'd be glad to help w/ this. Maybe @rajasegar if you check in a classic 3.16/3.18 app w/ the tests working I can pull and generate the pods artifacts and push a commit w/ the updated fixture and tests?